src/tests/base.py

```diff
@@ -398,7 +398,37 @@ def assertTmpFileNotTouchedBy(self, filename: str, func: Callable,
                                  relpath(filename, self.tmpd.name))
 
 
-class CleanAWSLoginEnvironment(TempDir):
+class CleanEnvironment(unittest.TestCase):
+    """Mixin Class for clearing and setting environment variables."""
```

Comment:
Interesting code. I don't take as much advantage of mixin classes as I would like.